frontend/app/components/banner/BannerBuilder.tsx

```diff
@@ -0,0 +1,332 @@
+import {
+  BANNER_FONT_SIZES,
+  FONT_FAMILY_OPTIONS,
+  SLIDE_ANIMATION,
+  type SlideAnimationType,
+  type BannerConfig
+} from '@shared/types'
+import {
+  BannerPositionSelector,
+  BannerColorsSelector,
+  Divider,
+  Checkbox,
+  TextareaField,
+  ToolsDropdown,
+  Slider,
+  CornerRadiusSelector,
+  Thumbnail
+} from '@/components'
+import type { ContentConfig } from '~/components/redesign/components/ContentBuilder'
+import type { AppearanceConfig } from '~/components/redesign/components/AppearanceBuilder'
+import BuilderAccordion from '~/components/redesign/components/BuilderAccordion'
+import { useUI } from '~/stores/uiStore'
+import { TitleInput } from '~/components/redesign/components/builder/TitleInput'
+import { DescriptionInput } from '~/components/redesign/components/builder/DescriptionInput'
+import { SectionHeader } from '~/components/redesign/components/SectionHeader'
+import {
+  SVGAnimation,
+  SVGColorPicker,
+  SVGHeaderPosition,
+  SVGRoundedCorner,
+  SVGText,
+  SVGThumbnail
+} from '~/assets/svg'
+import wmLogo from '~/assets/images/wm_logo_animated.svg?url'
+import { useState } from 'react'
+import { toolState } from '~/stores/toolStore'
+
+interface Props {
+  onRefresh: (section: 'content' | 'appearance') => void
+}
+
+const config: ContentConfig & AppearanceConfig = {
+  suggestedTitles: [
+    'How to support?',
+    'Fund me',
+    'Pay as you browse',
+    'Easy donate',
+    'Support my work'
+  ],
+  titleHelpText: 'Strong message to help people engage with Web Monetization',
+  titleMaxLength: 60,
+  messageLabel: 'Banner message',
+  messagePlaceholder: 'Enter your banner message...',
+  messageHelpText: 'Strong message to help people engage with Web Monetization',
+  messageMaxLength: 300,
+
+  showThumbnail: true,
+  fontSizeRange: BANNER_FONT_SIZES
+}
+
+export function BannerBuilder({ onRefresh }: Props) {
+  return (
+    <>
+      <ContentBuilder onRefresh={onRefresh} />
+      <AppearanceBuilder onRefresh={onRefresh} />
+    </>
+  )
+}
+
+function ContentBuilder({ onRefresh }: Props) {
+  const { actions: uiActions, state: uiState } = useUI()
+  const profile = toolState.currentConfig as BannerConfig
+
+  return (
+    <BuilderAccordion
+      title="Content"
+      isComplete={uiState.contentComplete}
+      onToggle={(isOpen) => {
+        uiActions.setActiveSection(isOpen ? 'content' : null)
+        if (isOpen) {
+          uiActions.setContentComplete(true)
+        }
+      }}
+      onRefresh={() => onRefresh('content')}
+      onDone={() => {
+        uiActions.setContentComplete(true)
+      }}
+      initialIsOpen={uiState.activeSection === 'content'}
+    >
+      <div className="flex flex-col gap-lg">
```

Comment:
@DarianM Can you check if we do need it (`flex flex-col gap-lg`)? It was in ContentBuilder, but not in ApperanceBuilder.